README.md

```diff
@@ -0,0 +1,53 @@
+Digital Marketplace Cloudwatch to Graphite
+=========================
+
+## Purpose
+
+Ships Cloudwatch metrics to Hosted Graphite using https://github.com/crccheck/cloudwatch-to-graphite.
+
+An example config file is also included (`config.yaml.example`) to help you define the correct structure for your metrics.
+
+## Initial setup
+
+#### Setup a virtualenv, activate it and install python dependencies with pip
+
+```
+virtualenv venv
```

Comment:
We should change this instruction to specify python3 as it's running in python 3 on the PaaS